### ISSUE TITLE
"declared but never defined" build warning/error on Fedora 22 (gcc 5.1.1)

### ISSUE THREAD

**Issue by tonycurtis** [openucx/ucx#343]
```
  CC       sm/mm/libuct_la-mm_ep.lo
In file included from /home/tc/src/ucx/master/build/../source/src/uct/sm/mm/mm_ep.h:11:0,
                 from /home/tc/src/ucx/master/build/../source/src/uct/sm/mm/mm_ep.c:8:
/home/tc/src/ucx/master/build/../source/src/uct/sm/mm/mm_iface.h:102:21: error: inline function 'uct_mm_flush' declared but never defined [-Werror]
 inline ucs_status_t uct_mm_flush();
                     ^
cc1: all warnings being treated as errors
Makefile:1086: recipe for target 'sm/mm/libuct_la-mm_ep.lo' failed
make[2]: *** [sm/mm/libuct_la-mm_ep.lo] Error 1
make[2]: Leaving directory '/home/tc/src/ucx/master/build/src/uct'
Makefile:622: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/home/tc/src/ucx/master/build'
Makefile:497: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by shamisp** [comment]
@tonycurtis - patch :-) ?

**Comment by shamisp** [comment]
fixed in #344
